Directory.Build.props

```
@@ -1,25 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Project>
   <PropertyGroup>
-    <NetVersion>net8.0</NetVersion>
-    <MauiPackageVersion>8.0.93</MauiPackageVersion>
+    <NetVersion>net9.0</NetVersion>
+    <MauiPackageVersion>9.0.0-rc.2.24503.2</MauiPackageVersion>
     <MauiCommunityToolkitPackageVersion>9.1.1</MauiCommunityToolkitPackageVersion>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Reminder to revert back to `latest` before merging

```suggestion
    <LangVersion>latest</LangVersion>
```

---

latest?

---

You're not wrong, but  I've updated all of the `[ObservableProperty]`s in the sample app to use partial properties. 

The MVVM toolkit uses the new `field` keyword for their implementations of partial properties, and in C# 13 we're still required to use `<LangVersion>preview</LangVersion>`  in order to access the new `field` keyword. 

Mads mentioned today that he plans on removing the `preview` requirement for the `field` keyword in C# 14.